test/lib/jdk/test/lib/util/CoreUtils.java

```diff
@@ -164,6 +188,10 @@ public static String getCoreFileLocation(String crashOutputString) throws IOExce
         throw new RuntimeException("Couldn't find core file location in: '" + crashOutputString + "'");
     }
 
+    public static String getCoreFileLocation(String crashOutputString) throws IOException {
+        return getCoreFileLocation(crashOutputString, -1);
+    }
```

Comment:
In order to be consistent, it seems that all users of `getCoreFileLocation()` should now be passing in the pid and we should get rid of this version that doesn't require it. That means they also all need to be converted to use `ProcessTools.executeProcessPreservePID()`.

Reply:
I pushed a new change that updates all tests.